assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -110,6 +110,14 @@
 
   @Override
   protected void configure() {
+    // Workaround for https://github.com/fabric8io/kubernetes-client/issues/2212
+    // OkHttp wrongly detects JDK8u251 and higher as JDK9 which enables Http2 unsupported for JDK8.
+    // Can be removed after upgrade to Fabric8 4.10.2 or higher or to Java 11
+    if (System.getProperty("java.version", "").startsWith("1.8")
+        && System.getenv("HTTP2_DISABLE") == null) {
```

Comment:
IMHO we should not make this configurable, because it is just plainly not supported under java8 if I am not mistaken.

Setting the system property makes it non-overridable by env var.

Reply:
@metlos want me to make this call `System.setProperty("http2.disable", "true")` permanent and not configurable?